app/lib/hooks/useIsVoipCallActive.ts

```
@@ -0,0 +1,6 @@
+import { useCallStore } from '../services/voip/useCallStore';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)